### PR TITLE
Add links to moosedocs tagging

### PR DESCRIPTION
**Pull request by GiudGiud**
refs #25387 

@cticenhour

### CODE REVIEW

File: python/MooseDocs/extensions/tagging.py [hunk 1]
```diff
@@ -98,6 +98,22 @@ def postExecute(self):
...
+                    if (entry == 'path'):
+                        path_value = tag_dict_str.split("path: ")[1].split(',')[0]
+
+                        # Split at the first modules/ (usually moose/modules/)
+                        if 'modules/' in path_value:
+                            path_value = path_value.split('modules/', 1)[1]
+                        else:
+                            # splitting at content is dangerous, same named paged within two different modules/xxx/doc/content
+                            # could be mixed together
+                            path_value = path_value.split('content/')[1]
```
Comment (cticenhour) [inline python/MooseDocs/extensions/tagging.py:111]: To make this less dangerous, we might try using `self.translator.findPages` here to check for possible page duplicates. See `Translator.py:151`.

**Review by cticenhour — COMMENTED** [review]
An initial review, pending check of the rendered documentation as well.

File: python/MooseDocs/extensions/tagging.py (hunk 1 above)
Comment (GiudGiud) [reply python/MooseDocs/extensions/tagging.py:111]: lemme try

Comment (GiudGiud) [reply python/MooseDocs/extensions/tagging.py:111]: umm findPages isnt working great. Not finding path_value and not finding its original pre-cut location. 

content/ will work great for the VTB, and modules/ will work great for modules pages (unless they refer to other modules)

Comment (cticenhour) [reply python/MooseDocs/extensions/tagging.py:111]: Sounds good - just wanted to see if that made things more robust. Thanks for trying!

**Comment by GiudGiud** [comment]
ah looks broken now
https://mooseframework.inl.gov/docs/PRs/25607/site/modules/electromagnetics/filter/index.html

**Comment by GiudGiud** [comment]
should be fixed

**Comment by GiudGiud** [comment]
Looks so good

Test failure unrelated

**Review by cticenhour — APPROVED** [review]

**Pull request merged**